pkg/contracts/solana/gateway_message_test.go

```diff
@@ -47,3 +47,24 @@ func Test_MsgWhitelistHash(t *testing.T) {
 		require.True(t, bytes.Equal(hash[:], wantHashBytes))
 	})
 }
+
+func Test_MsgWithdrawSPLHash(t *testing.T) {
+	t.Run("should pass for archived inbound, receipt and cctx", func(t *testing.T) {
+		// #nosec G115 always positive
```

Comment:
Let's use the ARRANGE-ACT-ASSERT pattern we have in comments in other tests. Same for other tests in the PR